storage/perfschema/table_variables_info.cc

```diff
@@ -88,7 +88,7 @@ PFS_engine_table *table_variables_info::create(PFS_engine_table_share *) {
 ha_rows table_variables_info::get_row_count(void) {
   mysql_mutex_lock(&LOCK_plugin_delete);
 #ifndef NDEBUG
-  extern mysql_mutex_t LOCK_plugin;
+  extern mysql_mutex_t LOCK_plugin [[maybe_unused]];
```

Comment:
PS code does not have `[[maybe_unused]]` attribute. Why do we need it for PXC?
It should be in WITH_WSREP block.

Reply:
I didn't check why it doesn not fail on PS. But it failed with errors on PXC though.
```
/home/venki/work/pxc/refresh/80/storage/perfschema/table_variables_info.cc: In static member function ‘static ha_rows table_variables_info::get_row_count()’:
/home/venki/work/pxc/refresh/80/storage/perfschema/table_variables_info.cc:91:24: error: unused variable ‘LOCK_plugin’ [-Werror=unused-variable]
   extern mysql_mutex_t LOCK_plugin;
                        ^~~~~~~~~~~
/home/venki/work/pxc/refresh/80/storage/perfschema/table_session_variables.cc: In static member function ‘static ha_rows table_session_variables::get_row_count()’:
/home/venki/work/pxc/refresh/80/storage/perfschema/table_session_variables.cc:94:24: error: unused variable ‘LOCK_plugin’ [-Werror=unused-variable]
   extern mysql_mutex_t LOCK_plugin;
                        ^~~~~~~~~~~
/home/venki/work/pxc/refresh/80/storage/perfschema/table_variables_by_thread.cc: In static member function ‘static ha_rows table_variables_by_thread::get_row_count()’:
/home/venki/work/pxc/refresh/80/storage/perfschema/table_variables_by_thread.cc:105:24: error: unused variable ‘LOCK_plugin’ [-Werror=unused-variable]
   extern mysql_mutex_t LOCK_plugin;
                        ^~~~~~~~~~~

```